### ISSUE TITLE
apache-common.conf _apache_error_client needs updating

### ISSUE THREAD

**Issue by Sarke** [fail2ban/fail2ban#1527]
The default `_apache_error_client` when installed was:

```
_apache_error_client = \[[^]]*\] \[(error|\S+:\S+)\]( \[pid \d+:\S+ \d+\])? \[client <HOST>(:\d{1,5})?\]
```

In `apache-common.conf` it explains the difference between Apache 2.2 and 2.4 log formats, and links to https://github.com/fail2ban/fail2ban/issues/268:

```
2.2: [Sat Jun 01 11:23:08 2013] [error] [client 1.2.3.4]
2.4: [Thu Jun 27 11:55:44.569531 2013] [core:info] [pid 4101:tid 2992634688] [client 1.2.3.4:46652]
```

However, my log format was slightly different still, and I have not changed it.  Here's an example of how my log looks and wasn't matched by the default:

```
[Sun Aug 28 01:55:21.359869 2016] [:error] [pid 22622] [client 83.251.237.86:54625] script '/var/www/wp-login.php' not found or unable to stat
```

The issue being that instead of `[error]` it is now `[:error]`, and the `:tid` was not included.  So, I suggest the following, which works for me:

```
_apache_error_client = \[[^]]*\] \[(error|\S*:\S+)\]( \[pid \d+(:\S+ \d+)?\])? \[client <HOST>(:\d{1,5})?\]
```

Server version: Apache/2.4.7 (Ubuntu)
Server built:   Jun 29 2016 13:21:39

**Comment by c-mertes** [comment]
I also installed apache and fail2ban on an ubuntu 14.04 virtual box and had the same problem.
The regex above fixed also my problem.

**Comment by sebres** [comment]
fixed long time ago (see 5cfe1081865cb9cd413cd252c20e4ae228163169)
